src/Microsoft.AspNet.OData.Shared/PatchMethodHandler.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.OData
+{    
+    /// <summary>
+    /// Base Interface for PatchMethodHandler
+    /// </summary>
+    public interface IPatchMethodHandler
```

Comment:
>IPatchMethodHandler [](http://example.com/codeflow?start=21&length=19)

I'm sure I asked this before, but why do we need an empty marker interface?  Can you add to the comments so that it's clear to others in the future?

Also, this should be in a separate file so that it is easy to find.